cli/cmd/silence_import.go

```diff
@@ -31,20 +36,29 @@ amtool silence import foo.json
 JSON data can also come from stdin if no param is specified.`
 }
 
+func addSilence(silence *types.Silence) (string, error) {
+	client, err := api.NewClient(api.Config{Address: (*alertmanagerUrl).String()})
```

Comment:
I think most of this function can disappear. It makes sense to me to create the client + silence client once, and then pass the silence client into each `addSilenceWorker` function. A bulk import could potentially have thousands of silences, which in its current form would create thousands of clients when only one is needed.

With this, the line `silenceID, err := addSilence(s)` in `addSilenceWorker` could just be `silenceID, err := silenceAPI.Set(context.Background(), *silence)`